app/src/scripts/user/user.store.js

```diff
@@ -423,6 +423,29 @@ let UserStore = Reflux.createStore({
         }
       })
   },
+
+  generateThumbnail(profile) {
```

Comment:
The user store is an odd place for this. It doesn't interact with the data in the store. It really looks like it should be a component named <AvatarThumbnail> or similar.